OsmAnd/res/values/strings.xml

```diff
@@ -661,9 +661,9 @@
     <string name="srtm_purchase_header">To see contour lines on the map, you need to buy and install the \'Contour Lines\' plugin</string>
     <string name="srtm_color_scheme">Color scheme</string>
     <string name="show_from_zoom_level">Display from zoom level</string>
-    <string name="routing_attr_allow_private_name">Allow private access</string>
+    <string name="routing_attr_allow_private_name">Allow trespassing</string>
     <string name="routing_attr_allow_private_description">Allow access to private areas.</string>
-    <string name="display_zoom_level">Display zoom level: %1$s</string>
+    <string name="display_zoom_level">Zoom level: %1$s</string>
```

Comment:
I think this may be a change I have attempted before, and can't remember the reasoning behind.